docs/migration/v4-migration.md

```diff
@@ -7,6 +7,7 @@ Chart.js 4.0 introduces a number of breaking changes. We tried keeping the amoun
 ### Charts
 
 * Charts don't override the default tooltip callbacks, so all chart types have the same-looking tooltips.
+* Scales options with the same first letter in the name don't override each other (e.g. `x2` will not override `x`).
```

Comment:
Think this is a bit clearer, it almost read to me that currently `x`, `x2` and `x3` would only have x3 but thats my personal interpertation

```suggestion
* Default scale override has been removed if the configured scale starts with `x`/`y`. Defining `xAxes` in your config will now create a second scale instead of overriding the default `x` axis
```